src/facade/facade.cc

```diff
@@ -44,8 +44,12 @@ ConnectionStats& ConnectionStats::operator+=(const ConnectionStats& o) {
   return *this;
 }
 
+ReplyStats::ReplyStats(ReplyStats&& other) noexcept {
+  *this = other;
```

Comment:
You define a move constructor yet you `copy assign here`. (The absl::hash is the only heavy weight utiltiy here) but i guess it's fine